### PR TITLE
DBZ-2250 Allowing to use CloudEventsConverter with Avro via Apicurio;

### PR DESCRIPTION
**Pull request by gunnarmorling**
https://issues.redhat.com/browse/DBZ-2250

### CODE REVIEW

File: debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/ApicurioRegistryTest.java [hunk 1]
```diff
@@ -48,7 +49,7 @@
  */
 public class ApicurioRegistryTest {
 
-    private static final String DEBEZIUM_VERSION = "1.2.0.CR1";
+    private static final String DEBEZIUM_VERSION = "1.2";
```
Comment (gunnarmorling) [inline debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/ApicurioRegistryTest.java:52]: That's chicken-and-egg issue here. I built the image with my core changes locally to verify the test passes. We probably would have to disable this test until the next release, containing the fix, is out.

Comment (jpechane) [reply debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/ApicurioRegistryTest.java:52]: @gunnarmorling I recommend you to create a special tag like `1.2-devel` and push the image to the registry so the test is passing also on CI. Then there will be a follow-up task for 1.2.0.Final/1.3 to point it elsewhere.

Comment (gunnarmorling) [reply debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/ApicurioRegistryTest.java:53]: Done; and logged https://issues.redhat.com/browse/DBZ-2256 for reverting it.

**Comment by gunnarmorling** [comment]
@jpechane, force-pushed an update. I'm driving the selection now based on the provided registry URL property. Addresses the test issue and also the concern you had.

File: debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/ApicurioRegistryTest.java (hunk 1 above)
Comment (gunnarmorling) [reply debezium-testing/debezium-testing-testcontainers/src/test/java/io/debezium/testing/testcontainers/ApicurioRegistryTest.java:53]: Pushed an update to the image as per my latest change.

**Comment by gunnarmorling** [comment]
CI green 🎉 !

**Comment by jpechane** [comment]
@gunnarmorling And applied, thanks!

**Pull request merged**